app/controllers/citizens/student_finances_controller.rb

```diff
@@ -1,27 +1,38 @@
 module Citizens
   class StudentFinancesController < CitizenBaseController
     def show
-      @form = LegalAidApplications::StudentFinanceForm.new(model: legal_aid_application)
+      @form = ::StudentFinances::AnnualAmountForm.new(model: irregular_income)
```

Comment:
Is this correct with nothing before the double colon? All the tests are passing but not seen this syntax before.